src/plugins/visualizations/public/visualize_app/components/visualize_editor.tsx

```diff
@@ -68,6 +68,7 @@ export const VisualizeEditor = ({ onAppLeave }: VisualizeAppProps) => {
   );
 
   const editorName = savedVisInstance?.vis.type.title.toLowerCase().replace(' ', '_') || '';
+  // TODO: Remove this comment after testing
```

Comment:
I've checked this and noticed some subtle different `pageName` sometimes.
Nothing major, just these few issues:
* `application:visualize` and `application:visualize:editor` are used for similar flows
* `application:visualize:editor:xxxxx` is triggered only on navigating back to dashboard from editor, not on enter